src/components/GrossProfit/MonthSelects.tsx

```diff
@@ -0,0 +1,48 @@
+import { ChangeEvent, FunctionComponent } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { MONTHS, YEARS } from '../../constants';
+import { updateActiveMonth, updateActiveYear } from '../../pages/GrossProfit/grossProfitSlice';
+
+const MonthSelects: FunctionComponent = () => {
```

Comment:
This component should be further divided into two sub-components: one for selecting months, the other one for selecting years